src/modm/board/raspberrypi_4/module.lb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Erik Henriksson
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+def init(module):
+    module.name = ":board:raspberrypi-4"
```

Comment:
Why (only) Raspberry Pi 4? As far as I can see this should also work on Pi 1, 2 and 3.

Reply:
The pin mapping / peripherals is different so if we want to extend the board with that then they would need to be separate.

Obviously you can still use this board on a RPi 1/2/3 but then you are on your own if it breaks in the future :)

Reply:
But inside this pull request I don't see any specialization regarding the RPi 4, therefore I favour a generic name for this module.

Reply:
Renamed it to "raspberrypi"